package.json

```diff
@@ -21,31 +21,35 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --timeout 15000 --require test/init.js",
+    "test": "mocha --timeout 180000  test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
```

Comment:
We prefer to keep mocha options in `test/mocha.opts` or `.mocharc.json` (see https://boneskull.com/mocha-v6/#introducing-mocharc-whatever), so that it's possible to run mocha for a subset of files without having to repeat all options.

For example, one may want to run juggler v4 tests only to get the results faster by not waiting for juggler v3 tests:

```sh
$ npx mocha node_modules/juggler-v4/test.js
```